### ISSUE TITLE
Javadoc comments are linted wrongly

### ISSUE THREAD

**Issue by gregorybleiker** [palantir/tslint#3781]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.6.2
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
/**
 * @param test
 * @returns void
 */
export function Test(test) {
...
```

with `tslint.json` configuration:

```json
  "whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-operator",
      "check-separator",
      "check-type"
    ]
```

#### Actual behavior
There is an indication of a missing whitespace between @param and test
#### Expected behavior
Javadoc comment should not be linted. If I disable the check-separator, it's ok.
